fineract-loan/src/main/java/org/apache/fineract/portfolio/loanaccount/domain/LoanSummary.java

```diff
@@ -38,6 +38,12 @@
 public class LoanSummary {
 
     // derived totals fields
+    @Column(name = "total_principal_derived", scale = 6, precision = 19)
+    private BigDecimal totalPrincipal;
```

Comment:
We might wanna add these fields to the `fineract-avro-schemas/src/main/avro/loan/v1/LoanAccountDataV1.avsc` so they goes in the business events as well